apps/challenges/models.py

```diff
@@ -98,7 +118,7 @@ class ChallengePhase(TimeStampedModel):
     challenge = models.ForeignKey('Challenge')
     is_public = models.BooleanField(default=False)
     is_submission_public = models.BooleanField(default=False)
-    test_annotation = models.FileField(upload_to=RandomFileName("test_annotations"))
+    test_annotation = models.FileField(upload_to=RandomFileName("test_annotations"), default=False)
```

Comment:
Can you explain me why default=False is required here?

Moreover if this is required, I will prefer that you create a separate PR for it